### PR TITLE
[ADD] coopiteasy_custom: Do not make closed projects selectable in tasks

### PR DESCRIPTION
**Pull request by carmenbianca**
## Odoo task (if applicable)

https://gestion.coopiteasy.be/web#id=8780&action=475&active_id=210&model=project.task&view_type=form&menu_id=536

## Checklist before approval

- [ ] Tests are present (or not needed).
- [ ] Credits/copyright have been changed correctly.
- [ ] Change log snippet is present.
- [ ] (If a new module) Moving this to OCA has been considered.

### CODE REVIEW

File: coopiteasy_custom/views/project_task.xml [hunk 1]
```diff
@@ -14,13 +14,18 @@
...
+                >[("active", "=", True), ("project_status.is_closed", "=", False)]</attribute>
+            </xpath>
             <xpath
                 expr="//field[@name='timesheet_ids']/tree/field[@name='project_id']"
                 position="attributes"
             >
                 <attribute
                     name="domain"
-                >[("project_status.is_closed", "=", False)]</attribute>
+                >[("active", "=", True), ("project_status.is_closed", "!=", False)]</attribute>
```
Comment (huguesdk) [inline coopiteasy_custom/views/project_task.xml]: i don’t understand why this has to change, especially inverting the `is_closed` condition.

File: coopiteasy_custom/views/project_task.xml [hunk 2]
```diff
@@ -14,13 +14,18 @@
             ref="hr_timesheet_task_change_project.project_task_form_view"
         />
         <field name="arch" type="xml">
+            <xpath expr="//group/field[@name='project_id']" position="attributes">
+                <attribute
+                    name="domain"
+                >[("active", "=", True), ("project_status.is_closed", "=", False)]</attribute>
```
Comment (huguesdk) [inline coopiteasy_custom/views/project_task.xml]: why `active = True`? this is by default.

**Review by huguesdk — CHANGES_REQUESTED** [review]
would you please add a comment before the first `xpath` and move the current comment before the 2nd `xpath`?

File: coopiteasy_custom/views/project_task.xml (hunk 1 above)
Comment (carmenbianca) [reply coopiteasy_custom/views/project_task.xml]: inverted condition was a typo :x fixed

File: coopiteasy_custom/views/project_task.xml (hunk 2 above)
Comment (carmenbianca) [reply coopiteasy_custom/views/project_task.xml]: I got the `active = True` condition from upstream's domain on project_id, but it appears to do exactly nothing, so I've reverted that change.

**Review by huguesdk — APPROVED** [review]
lgtm! 🚀 extra bonus if you change the comment as explained in my previous review. 😉

**Pull request merged**